src/app.module.ts

```diff
@@ -3,8 +3,22 @@ import { UserModule } from './user/user.module';
 import { AuthModule } from './auth/auth.module';
 import { RecipeModule } from './recipe/recipe.module';
 import { ConfigModule } from '@nestjs/config';
+import { CacheModule } from '@nestjs/cache-manager';
+import * as redisStore from 'cache-manager-redis-store';
 
 @Module({
-  imports: [UserModule, AuthModule, RecipeModule, ConfigModule.forRoot()],
+  imports: [
+    UserModule,
+    AuthModule,
+    RecipeModule,
+    ConfigModule.forRoot(),
+    CacheModule.register({
+      isGlobal: true,
+      store: redisStore,
+      host: process.env.REDIS_HOST,
+      port: process.env.REDIS_PORT,
+      ttl: 30,
```

Comment:
please double-check the units of ttl those are seconds or milliseconds? (see the notes regarding that in nestjs docs)